src/aleph/vm/pool.py

```diff
@@ -58,6 +62,10 @@ def __init__(self):
         logger.debug("Initializing SnapshotManager ...")
         self.snapshot_manager.run_snapshots()
 
+        logger.debug("Loading existing executions ...")
+        loop = asyncio.get_event_loop()
+        loop.run_until_complete(self._load_persistent_executions())
```

Comment:
Why not use the simpler `asyncio.run()` ?

```suggestion
        asyncio.run(self._load_persistent_executions())
```